index.js

```diff
@@ -637,6 +644,42 @@ var commands = {
 				d.conn.send("error:You're not in a room.");
 			}
 		}
+	},
+	pause: {
+		data: true,
+		handler: function(d) {
+			for(var id in games) {
+				 var gd = games[id];
+				 var ind = gd.users.indexOf(d.user);
+					if(ind>-1) {
+						for(var id in games) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["id" is already defined](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2691216238/issues/source?bid=3248399&fileBranchId=3339424#l655)